src/components/AddElement.tsx

```diff
@@ -13,10 +13,13 @@ const PressableContainer = styled(PressableOpacity)`
 const AddCustomDisciplineText = styled(SubheadingPrimary)`
   text-transform: uppercase;
   padding-left: ${props => props.theme.spacings.xs};
+  margin-right: ${props => props.theme.spacings.xs};
 `
 
 const Explanation = styled(ContentSecondary)`
   padding: 0 ${props => props.theme.spacings.xxl} 0 11%;
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  color: ${props => props.theme.colors.textSecondary};
```

Comment:
You don't need to set the color to `textSecondary` as your StyledComponent extends the `ContentSecondary`-component, which already has set the color to `textSecondary`.

Have a look at `Content.tsx` and please add there a new item ContentSecondaryBold, which should be similar to `ContentSecondaryLight`. Then use this new item here.

Reply:
Should I remove then font-family from Explanation component? Because there is already in **ContentSecondaryBold**